### ISSUE TITLE
Support relative nodePath settings 

### ISSUE THREAD

**Issue by oBusk** [Microsoft/vscode-tslint#264]
| What? | Version?
| - | - |
| VScode | `1.16.1` |
| vscode-tslint |  `1.0.7` |
| typescript | `2.5.2` |
| TSLint | `5.7.0` | 

Running windows 10.

After updating to VSCode 1.16, TSlint plugin stopped working.

Whenever I open a `.ts`-file i get 

```
To use TSLint in this workspace please install tslint using 'npm install tslint' or globally using 'npm install -g tslint'.
TSLint has a peer dependency on `typescript`, make sure that `typescript` is installed as well.
You need to reopen the workspace after installing tslint.
```

Our `settings.json` looks like this:

```json
{
    "typescript.tsdk": "./scripts/gulp/node_modules/typescript/lib",
    "tslint.nodePath": "./scripts/gulp/node_modules/tslint/lib",
    "tslint.jsEnable": true,
    "tslint.exclude": [
        "**/WebApplication/DotCom.WebApplication/app/lib/**",
        "**/WebApplication/DotCom.WebApplication/app/prototype/lib/**"
    ]
}
```

It's worth noting that the structure of our project is as follows:

```
ProjectRoot (Workspace root)
├─ .vscode
│  └─ settings.json
├─ Scripts
│  └─ gulp
│     ├─ gulpfile.js
│     └─ node_modules
└─ WebApplication
   └─ DotCom.Webapplication
      └─ app
         ├─ tsconfig.json
         ├─ code.ts
         └─ subComponent
            └─ subComponent
```

Installing tslint globally makes it work (using the globally installed tslint)

**Comment by egamma** [comment]
Can you try whether this works when the nodePath setting is using an absolute path?

Support for relative paths got broken in vscode-tslint 1.0.7.

**Comment by oBusk** [comment]
Yes it seems to work with an absolute path!

**Comment by egamma** [comment]
fixed in vscode-tslint version 1.0.8.
